automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/methodrepo/MethodRepoSpec.scala

```diff
@@ -9,7 +9,7 @@ import org.scalatest._
 
 class MethodRepoSpec extends FreeSpec with WebBrowserSpec with Matchers with CleanUp {
 
-  implicit val authToken: AuthToken = AuthTokens.hermione
+  implicit lazy val authToken: AuthToken = AuthTokens.hermione
```

Comment:
Out of curiosity, I noticed a lot of these in a previous PR. Is there a functional reason for this? It didn't seem worth asking when mixed in with other changes but now that this is on its own I want to know.